### PR TITLE
Delete Ihr/ Ihres/ Ihnen

### PR DESCRIPTION
**Pull request by CV113**
no "Ihr"/"Ihren" -> because its a family members certificate (please see also the figma screens COVID19_Master_2.18 – Figma)
--> https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12656

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -776,19 +776,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
-"ExposureSubmissionResult_testInvalidDesc" = "Es gab ein Problem bei der Auswertung Ihres Tests. Bitte kontaktieren Sie das Testcenter oder das zuständige Labor, um Informationen zum weiteren Vorgehen zu erhalten.";
+"ExposureSubmissionResult_testInvalidDesc" = "Es gab ein Problem bei der Auswertung des Tests. Bitte kontaktieren Sie das Testcenter oder das zuständige Labor, um Informationen zum weiteren Vorgehen zu erhalten.";
 
 "ExposureSubmissionResult_testExpired" = "Test kann nicht ausgewertet werden";
 
-"ExposureSubmissionResult_testExpiredDesc" = "Es gab ein Problem bei der Auswertung Ihres Tests. Ihr QR Code ist bereits abgelaufen.";
+"ExposureSubmissionResult_testExpiredDesc" = "Es gab ein Problem bei der Auswertung des Tests. Ihr QR Code ist bereits abgelaufen.";
 
-"ExposureSubmissionResult_testPending" = "Ihr Testergebnis liegt noch nicht vor";
+"ExposureSubmissionResult_testPending" = "Das Testergebnis liegt noch nicht vor";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:785]: "Ihr" is correct here; there is an extra string for pending test results for family members

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -776,19 +776,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
 
 "ExposureSubmissionResult_familyMember_testPending" = "Ihr Testergebnis liegt noch nicht vor";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:787]:
```suggestion
"ExposureSubmissionResult_familyMember_testPending" = "Das Testergebnis liegt noch nicht vor";
```

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
Maybe all strings should be duplicated to reflect wether it's a test for the owner of the smartphone or a family member. Currently only some are duplicated to differentiate between these cases.

**Comment by CV113** [comment]
Closed
Changes for 2.22

**Pull request closed without merging**